### PR TITLE
Deploys static hosted version of loc-components

### PR DESCRIPTION
**Pull request by markuskobler**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -5,15 +5,12 @@
     <title>Localytics Component Library</title>
     <link rel="shortcut icon" type="image/x-icon" href="/assets/favicon.ico">
     <script type="text/javascript" src="//use.typekit.net/mgr1bnc.js"></script>
-    <script type="text/javascript">
-      try{Typekit.load();}catch(e){}
+    <script>
+     (function(d) { var config = { kitId: 'mgr1bnc', scriptTimeout: 3000, async: true }, h=d.documentElement,t=setTimeout(function(){h.className=h.className.replace(/\bwf-loading\b/g,"")+" wf-inactive";},config.scriptTimeout),tk=d.createElement("script"),f=false,s=d.getElementsByTagName("script")[0],a;h.className+=" wf-loading";tk.src='https://use.typekit.net/'+config.kitId+'.js';tk.async=true;tk.onload=tk.onreadystatechange=function(){a=this.readyState;if(f||a&&a!="complete"&&a!="loaded")return;f=true;clearTimeout(t);try{Typekit.load(config)}catch(e){}};s.parentNode.insertBefore(tk,s) })(document);
```
Comment (seancrater) [inline public/index.html]: Kit ID was old, new one is `pli3wny`

File: webpack.config.js [hunk 2]
```diff
@@ -62,11 +50,42 @@ module.exports = {
...
-  ],
   resolve: {
     extensions: ['.js', '.json', '.jsx', '.scss', '.css']
   }
 };
+
+
+module.exports = (env) => {
+  switch (env) {
+  case 'dev':
```
Comment (seancrater) [inline webpack.config.js]: Awesome use of a `switch`, should start doing this in my own configs...

File: .travis.yml [hunk 3]
```diff
@@ -1,11 +1,29 @@
...
-  - 6.10.1
+node_js: 6.10.1
 dist: trusty
-
 env:
+  matrix:
   - TEST_SUITE=cover
   - TEST_SUITE=lint
+  global:
+    secure: XdXWciRIyrpzWeNMhWA7XIJ2lx2KOI6J66C2vUYfe1YDNgJE6nt75FOb0vegrIWDlULgZ1X6jC6pZGRadKuvFDAa7vIJ36Hi4lD1Qu37mU4H6zhjFhOSepsgElxpKBkEMIbLC3kgVJ5R/X7yPOAeQSu7TXeRS5TW2rD7ogHSEbiaJk5bs9hfWcUV3Zy9U8X2wcAo5BEKkAnlAKT84Hz3gmS7+p58MoGzHHfTUG6ktAm7WT7NBqLsB8z7M4HykW+O0x8v0HyFZdYu/ptqPam2TDDZbt4hkbVHww1JKmyL6qglIDwP9L2m9QUSYZkY+ZQ6izwHc3UWn4WxLaCuzcWUv4SazZqyX7fcsnyGNPT+76vXK5kEuXsrIYyZi8+lfRI+rGRDF6kA+0SsaffKf7WOLnE56brJemyIRZV/ttPbQzcljpc65vFTVVLTB6/cl2DckonySBZrATyP8AjpHXBHbLYurGP+/MN7iUMB3WHI2n0K3bDXFPQlUx/q7JTgPUD8hnSrCmEBVyCDht6DpqIksPQERkne/pOHSBajMQnAP8ii3w9JCyNAy8+JPXTXJjjOcXtXtX49tF3baDXhgPc1rvIggudh+DHPGqXGliRN2LSKxqo3zSgxMP+n+B8tZD7Z20Ty7PS8rAQm/Jc7kHs92UxbxQJNY6g6yc/wglEzLPE=
```
Comment (meagonqz) [inline .travis.yml]: What user is this?

Comment (markuskobler) [inline .travis.yml]: I created a new user in sandbox with permissions to upload to this bucket only

**Pull request merged**